### ISSUE TITLE
importing addcopyfighandler raises ValueError with matplotlib 3.9.0 on Linux

### ISSUE THREAD

**Issue by haroldmeerwaldt** [joshburnett/addcopyfighandler#15]
Hi there,

I am doing an import of addcopyfighandler and I get the following error:
```
    from addcopyfighandler import copyfig, oldfig
/usr/local/lib/python3.10/site-packages/addcopyfighandler.py:262: in <module>
    raise ValueError(f'Unsupported matplotlib backend ({backend}) on this OS.')
E   ValueError: Unsupported matplotlib backend (qtagg) on this OS.
```

I am on Ubuntu:
Linux runner-XXX 5.4.0-176-generic #196-Ubuntu SMP Fri Mar 22 16:46:39 UTC 2024 x86_64 GNU/Linux

Most likely spelling the backend without a capital Q in matplotlib 3.9.0 causes the issue.

Could you have a look at this?

**Comment by joshburnett** [comment]
What version of Qt are you using? PyQt5, PyQt6, PySide2, or PySide6?

**Comment by joshburnett** [comment]
Never mind. I just checked and both PySide6 and PyQt6 will result in matplotlib reporting ‘QtAgg’ as their backend. Too bad the release notes for matplotlib don’t mention this particular change.

Also looks like this issue is across all OSes. Thanks for reporting the issue; I’ll fix it.

**Comment by joshburnett** [comment]
Fixed in 3.2.1.
